src/main/java/org/cyclops/integrateddynamics/core/logicprogrammer/ValueTypeListLPElement.java

```diff
@@ -302,7 +303,11 @@ public void drawGuiContainerBackgroundLayer(int guiLeft, int guiTop, TextureMana
 
         @Override
         public void onChanged() {
-            element.setListValueType(valueTypeSelector.getActiveElement());
+            IValueType newType = valueTypeSelector.getActiveElement();
+            element.setListValueType(newType);
+            if(arrowAdd != null) {
+                arrowAdd.enabled = newType != ValueTypes.CATEGORY_ANY;
```

Comment:
What does this do exactly?

Reply:
It disables the `+` button when you select to make a List<Any> in the logic programmer, so you can only use it to make empty lists directly. This is because I'm not sure of a good idea to let users build these kinds of lists directly.

Reply:
It makes sense IMO to allow users to create such lists. But this would require some significant changes in the editor, so I agree that it makes sense to leave this out-of-scope for this PR.

Reply:
Agreed.